src/utils/throttle.js

```diff
@@ -1,7 +1,7 @@
 export default (callback, wait, immediate = false) => {
   let timeout = null;
   let initialCall = true;
-
+  console.log("callback", callback); // eslint-disable-line
```

Comment:
Did you mean to leave this comment here?

Reply:
No. Forgot to delete it. Thanks. Fixed now.